Tests/IncludeFormatingTest.cs

```diff
@@ -46,7 +46,11 @@ public void Sorting_BlanksAfterRegexGroup()
 
 #include ""a.h""
 #include <b.hpp>
-#include <c.hpp>";
+#include <c.hpp>
+
+
+
+";
```

Comment:
You'll see a lot of empty lines in the "expected output", because every test except one removes duplicates, and there are a lot of duplicates.